internal/core/command/put_test.go

```diff
@@ -17,16 +17,21 @@ package command
 import (
 	"context"
 	"io/ioutil"
+	"net/http"
+	"strings"
 	"testing"
 
 	"github.com/edgexfoundry/go-mod-core-contracts/clients"
 	"github.com/edgexfoundry/go-mod-core-contracts/clients/logger"
+
+	"github.com/stretchr/testify/require"
 )
 
 func TestNewPutCommandWithCorrelationId(t *testing.T) {
```

Comment:
This looks like it is duplicated in the test `TestNewPutCommandContentType`. If so can we remove this?